packages/lib/src/vis/surface/SurfaceShaderMaterial.tsx

```diff
@@ -0,0 +1,25 @@
+import { DoubleSide } from 'three';
+
+const SHADER = {
+  vertexShader: `
+    varying vec3 vertexColor;
+
+    void main() {
+      gl_Position = projectionMatrix * modelViewMatrix * vec4( position, 1.0 );
+      vertexColor = color;
+    }
+  `,
+  fragmentShader: `
+    varying vec3 vertexColor;
+
+    void main() {
+      gl_FragColor = vec4(vertexColor.rgb, 1);
+    }
+  `,
+};
```

Comment:
Simplified version of the `GlyphMaterial` shader. 

Again, bound to change if I move to a `Heatmap`-like shader